source/jormungandr/jormungandr/interfaces/v1/test/journey_tests.py

```diff
@@ -51,10 +52,10 @@ def __init__(self):
 
         #and we will use a list of instances
         self.regions = {
-            'equador': MockInstance(True, 'equador'),
-            'france': MockInstance(False, 'france'),
-            'peru': MockInstance(False, 'peru'),
-            'bolivia': MockInstance(True, 'bolivia')
+            'equador': MockInstance(True, 'equador', 0),
+            'france': MockInstance(False, 'france', 0),
+            'peru': MockInstance(False, 'peru', 0),
+            'bolivia': MockInstance(True, 'bolivia', 0)
```

Comment:
In all the tests the value is the same, so it's not tested, can you add a test with different values ?